modules/administration-guide/examples/ref_che-supported-platforms-and-installation-methods.adoc

```diff
@@ -2,63 +2,27 @@
 //
 // supported_platforms
 
-The following section provides information about the availability of {prod-short} {prod-ver} on {platforms-name} infrastructures, and about their supported installation methods.
+The following section provides information about the availability of {prod-short} {prod-ver} on {platforms-name} infrastructures.
 
 {prod} can be installed on:
 
 * {kubernetes} infrastructures starting at version {kube-ver-min}
-* {ocp} versions 3.11, 4.3, and 4.4
+* {ocp} starting at version 4.8
 
-The following options are available:
+{prod} can be installed on all major Public Clouds such as:
 
-.Local installation on {kubernetes}
-[options="header",cols="2*.^"]
-|===
-|{kubernetes}
-|Installation method
-
-a|* `minikube`
-* `microk8s`
-* `docker-desktop`
-* `kind`
-|`{prod-cli}` using the Operator
-|===
-
-.Local installation on OpenShift
-[options="header",cols="2,2"]
-|===
-|{ocp}
-|Installation method
-
-|CodeReady Containers
-|OperatorHub
-|===
-
-.Public cloud installation on {platforms-name}
-[options="header",cols="25,25,25,25"]
-|===
-|
-|{kubernetes} +
-{kube-ver-min} or higher
-|{ocp} +
-3.11
-|{ocp} +
-4.3 and 4.4
-
-a|* Microsoft Azure
 * Amazon Web Services
 * Google Cloud
 * IBM Cloud
-.^|`{prod-cli}` using the Operator
-.^|`{prod-cli}` using the Operator
-.^|OperatorHub
-|===
+* Microsoft Azure
+* Rancher
+
+WARNING: Setting up [Users' Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) is required for deploying {prod-short} on {kubernetes} infrastructures. For {ocp} no additional setup is needed.
```

Comment:
You are using markdown links, you should use asciidoc instead.

Reply:
ack, used what was suggested in https://github.com/eclipse-che/che-docs/pull/2284#discussion_r849692962 ;-)

Reply:
fixed

Reply:
oh sorry, my fault 😄